airbyte-cdk/python/airbyte_cdk/sources/declarative/parsers/model_to_component_factory.py

```diff
@@ -117,7 +117,6 @@ def __init__(
         self._init_mappings()
         self._limit_pages_fetched_per_slice = limit_pages_fetched_per_slice
         self._limit_slices_fetched = limit_slices_fetched
-        self._emit_connector_builder_messages = emit_connector_builder_messages
```

Comment:
It feels more difficult to have to propagate `emit_connector_builder_messages` through every constructor in the this factory. 

Should we instead just leave this code as is and as part of `manifest_declarative_source.py:__init__()` we continue to pass `emit_connector_builder_messages` as a constructor parameter as we are already doing. In that file it would look something like this:
```
self._source_config = propagated_source_config
self._debug = debug
self._emit_connector_builder_messages = emit_connector_builder_messages
self._constructor = component_factory if component_factory else ModelToComponentFactory(emit_connector_builder_messages=True)
```

Reply:
done